erpnext/controllers/item_variant.py

```diff
@@ -51,7 +51,7 @@ def validate_item_variant_attributes(item, args):
 				frappe.throw(_("Increment for Attribute {0} cannot be 0").format(attribute))
 
 			is_in_range = from_range <= flt(value) <= to_range
-			precision = len(cstr(increment).split(".")[-1].rstrip("0"))
+			precision = max(len(cstr(value).split(".")[-1].rstrip("0")), len(cstr(increment).split(".")[-1].rstrip("0")))
```

Comment:
better way to write it would be:

`max((len(cstr(v).split(".")[-1].rstrip("0")) for v in (value, increment))`